ChaosMod/Effects/db/Peds/PedsInvisibleAxeJesus.cpp

```diff
@@ -0,0 +1,57 @@
+#include <stdafx.h>
+
+static void OnStart()
+{
+	static constexpr Hash modelHash = -835930287;
+
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	static const Hash playerGroup = GET_HASH_KEY("PLAYER");
+	static const Hash civGroup = GET_HASH_KEY("CIVMALE");

```

Comment:
probably shouldn’t attack everyone and just the player

Reply:
I just copied the griefer jesus code for that bit, seems to work just fine.